src/iceberg/test/CMakeLists.txt

```diff
@@ -83,7 +83,8 @@ add_iceberg_test(table_test
                  json_internal_test.cc
                  table_test.cc
                  schema_json_test.cc
-                 table_metadata_builder_test.cc)
+                 table_metadata_builder_test.cc
+                 pending_update_test.cc)
```

Comment:
These files are unfortunately not sorted alphabetically. Could you help fix it as well?

Reply:
sure